tilemap/Map.cs

```diff
@@ -1,21 +1,57 @@
 using Godot;
-using System;
 
 public class Map : Node2D
 {
     // Declare member variables here. Examples:
     // private int a = 2;
     private Player player;
+    private TileMap walls;
+    private bool hasKey;
+    
+    private int keyId;
+    private int doorId;
```

Comment:
i think this should be a dictionary and named like `TILES_IDS`:

```csharp
private Dictionary<string, int>() = new Dictionary<string, string>() { 
    "key": -1,
    "door": -1,
    // ...
 }
```

Reply:
[82b574f](https://github.com/fourlastor-jams/game-off-2022/pull/14/commits/82b574f0cb0238481d1df53c82422b3d8a95ffe9)